### PR TITLE
Add CI/CD to leancode_markup

### PR DESCRIPTION
**Pull request by pdenert**

### CODE REVIEW

File: .github/workflows/leancode_markup-publish.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    permissions:
+      id-token: write
+
+    defaults:
+      run:
+        working-directory: packages/leancode_markup
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```
Comment (shilangyu) [inline .github/workflows/leancode_markup-publish.yml:22]: should be v4

File: .github/workflows/leancode_markup-publish.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          sdk: 3.2
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '3.16.x'
+          cache: true
+          
+      - name: Publish
+        run: dart pub publish -f
```
Comment (shilangyu) [inline .github/workflows/leancode_markup-publish.yml:36]: I think we now publish using the mobile-tools script (see enhanced_gradients-publish.yml for reference)

File: .github/workflows/leancode_markup-test.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+name: leancode_markup test
+
+on:
+  push:
+    branches: [master]
+    ignore-tags: ['leancode_markup-v*']
```
Comment (shilangyu) [inline .github/workflows/leancode_markup-test.yml:6]: Should be `tags-ignore`

File: .github/workflows/leancode_markup-test.yml
Comment (shilangyu) [inline .github/workflows/leancode_markup-test.yml:1]: Shouldn't this be a flutter setup?

**Review by shilangyu — CHANGES_REQUESTED** [review]

**Comment by shilangyu** [comment]
Side note: leancode_markup is missing in root README

**Comment by pdenert** [comment]
> Side note: leancode_markup is missing in root README

I'm aware of that. I plan to add it after the package release

**Review by shilangyu — CHANGES_REQUESTED** [review]

File: .github/workflows/leancode_markup-test.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+      - name: Run analyzer
+        run: |
+          dart analyze
+          dart run custom_lint
+
+      - name: Run tests with coverage
+        run: |
+          dart test --coverage=coverage
+          dart run coverage:format_coverage --lcov --in=coverage --out=coverage/lcov.info --report-on=lib
```
[the remark this answers is not in the document]
Comment (pdenert) [reply .github/workflows/leancode_markup-test.yml]: I copied the coverage from cqrs-test.yml. I don't need it either

**Review by shilangyu — APPROVED** [review]

**Pull request merged**